tutorials-and-examples/inference-servers/jetstream/http-server/Dockerfile

```diff
@@ -4,7 +4,7 @@
 FROM ubuntu:22.04
 
 ENV DEBIAN_FRONTEND=noninteractive
-ENV JETSTREAM_VERSION=v0.2.0
+ENV PYTORCH_JETSTREAM_VERSION=v0.2.2
```

Comment:
This is very specific to the Pytorch jetstream version. This HTTP server not only interacts with Jetstream pytorch but also with MaxText

Reply:
Does it still break maxtext? There is a new release for maxtext - https://github.com/google/maxtext/releases/tag/jetstream-v0.2.2

Reply:
Ah it may be supported now then. Ill try rebuilding the http server image for both and see if they work. Will also revert back to using Jetstream repo rather than jetstream-pytorch in the dockerfile

Reply:
Thanks @francescov1, just to confirm, you were able to validate that the jetstream server works for MaxText and Jetstream-pytorch after bumping the versions?

Reply:
@vivianrwu Fixed. Tested jetstream pytorch and it works correctly. Havent been able to try maxtext cause running into an issue with the checkpoint conversion script, if you have a chance could you test it out?

Reply:
Thanks! Seems like https://github.com/GoogleCloudPlatform/ai-on-gke/pull/695 got merged though, could you rebase your changes, specifically adding this diff https://github.com/GoogleCloudPlatform/ai-on-gke/pull/678/files#diff-1fc2e925287a55595a2ac559a6341e09715cc28d6086a89f72e691495e26b394R85-R86. Then I'll approve for submission

Reply:
Great, done!